### PR TITLE
feat/storybook-7-upgrade-6

### PR DESCRIPTION
**Pull request by panvourtsis**

### CODE REVIEW

File: src/components/IconButton/IconButton.mdx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+### IconButton Types and Sizes
+
+IconButton with different types `primary, secondary, tertiary` and different shape types `circle, square` <br/>
+There are 2 IconButton sizes, `normal` and `compact`.
+
+<Canvas of={IconButtonStories.IconButtonTypes} />
+
+### Playground
+
+<Canvas of={IconButtonStories.IconButtonPlayground} />
```
Comment (kostasdano) [inline src/components/IconButton/IconButton.mdx:39]: can we rename this to Playground?

File: src/components/IconButton/IconButton.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+          'check'
+        )}
+        size={select('size', ['compact', 'normal'], 'normal')}
+        type={select('type', ['primary', 'secondary', 'tertiary'], 'primary')}
+        shape={select('shape', ['circle', 'square'], 'circle')}
+      />
+    </Stack>
+  ),
+
+  name: 'IconButton Playground',
```
Comment (kostasdano) [inline src/components/IconButton/IconButton.stories.tsx:77]: same here, creates inconsistency on storybook docs

File: src/components/IconButton/IconButton.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+      </Stack>
+      <Stack>
+        <IconButton name="dotsVertical" shape="square" size="compact" />
+        <IconButton name="dotsVertical" type="secondary" shape="square" size="compact" />
+        <IconButton name="dotsVertical" type="tertiary" shape="square" size="compact" />
+      </Stack>
+    </>
+  ),
+
+  name: 'IconButton Types',
```
Comment (kostasdano) [inline src/components/IconButton/IconButton.stories.tsx:56]: rename this to `IconButton Types and Sizes`

File: src/components/NumberField/NumberField.mdx [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+NumberField can have a stepper with increment and decrement buttons. Step defaults to 1 if not provided.
+
+<Tip>
+  [Accessibility] - Once the NumberField is focused, Stepper keyboard navigation is supported with
+  the arrowUp and arrowDown keys.
+</Tip>
+
+<Canvas of={NumberFieldStories.NumberFieldWithStepper} />
+
+## NumberField with min/max values
```
Comment (kostasdano) [inline src/components/NumberField/NumberField.mdx:63]: one more hashtag here please :P

**Review by kostasdano — APPROVED** [review]

File: src/components/IconButton/IconButton.stories.tsx (hunk 3 above)
Comment (panvourtsis) [reply src/components/IconButton/IconButton.stories.tsx]: fixed here https://github.com/Orfium/orfium-ictinus/pull/727/commits/75b0366b3ffd3b719788f9dda327dbe5aa2a898a

File: src/components/IconButton/IconButton.stories.tsx (hunk 2 above)
Comment (panvourtsis) [reply src/components/IconButton/IconButton.stories.tsx]: fixed here https://github.com/Orfium/orfium-ictinus/pull/727/commits/75b0366b3ffd3b719788f9dda327dbe5aa2a898a

File: src/components/IconButton/IconButton.mdx (hunk 1 above)
Comment (panvourtsis) [reply src/components/IconButton/IconButton.mdx]: fixed here https://github.com/Orfium/orfium-ictinus/pull/727/commits/75b0366b3ffd3b719788f9dda327dbe5aa2a898a

File: src/components/NumberField/NumberField.mdx (hunk 4 above)
Comment (panvourtsis) [reply src/components/NumberField/NumberField.mdx]: fixed here https://github.com/Orfium/orfium-ictinus/pull/727/commits/44a4ee7b42f5b95c2214b262176b85b1c7c913d6

**Pull request merged**